src/persistence.js

```diff
@@ -11,6 +11,11 @@ var Base64 = Polyfill.Base64,
 export default function _Persistence(mpInstance) {
     var self = this;
 
+    // QUESTION: Should isEnabled be a param on the constructor, or should it read from config?
+    this.isEnabled = function() {
```

Comment:
I think future state where we decouple persistence enough, then if persistence is not initialized, having it as a param on the contructor makes less sense.  for now, let's read from config and check on store.SDKConfig the way you are currently doing it.

Reply:
Based on our conversation, we will leave this `isEnabled` method here in case we have other checks to determine if Persistence is disabled for other reasons besides the config. For example, if local storage not exist.